pages/blog/[slug].tsx

```diff
@@ -32,41 +32,74 @@ const PostLayout: NextPage<{ post: Post }> = ({ post }) => {
         template={false}
       />
 
-      <BlogNav />
+      <Nav />
 
-      <article className="mx-auto py-8 lg:py-12 " id="content">
-        <div className="mx-auto mb-6 max-w-3xl px-4">
+      <article className="mx-auto mt-[74px] py-8 lg:py-12 " id="content">
+        <div className="mx-auto mb-8 max-w-5xl px-6">
+          <Link href="/blog" passHref>
+            <a>
+              <span className="tracking-[0]">&lt;-</span>
+              {` `}
+              {t(`back-to-list`)}
+            </a>
+          </Link>
+        </div>
+
+        <div className="mx-auto mb-6 max-w-5xl px-6">
           <Link href="/blog">
             <a className="text-sm font-semibold uppercase tracking-tight text-pinkRed">
               {post.category}
             </a>
           </Link>
 
-          <h1 className="mt-1 text-3xl font-bold lg:text-5xl">{post.title}</h1>
-          <p className=" prose mt-8 text-neutral-500 dark:prose-invert dark:text-neutral-400  sm:prose-lg">
+          <h1 className="mt-1 text-5xl font-bold tracking-tight lg:text-5xl">
+            {post.title}
+          </h1>
+
+          <p className=" prose mt-8 text-neutral-700 dark:prose-invert dark:text-neutral-300  sm:prose-lg">
             {post.summary}
           </p>
-          <div className="mt-8 flex items-center">
+
+          <a className="my-8 inline-flex h-[34rem] overflow-hidden rounded-md">
+            <Image
+              src={post.image}
+              width="1080"
+              height="540"
+              quality={100}
+              className="rounded-md object-cover object-center "
+              alt=""
+              priority
+            />
+          </a>
+          <div className="pb-2 text-neutral-600 dark:text-neutral-200">
+            Author
+          </div>
```

Comment:
I don't think this is required tbh

![image](https://user-images.githubusercontent.com/34704796/193829204-01e0bdcf-f99b-4950-9391-389fe687d200.png)

Updated:
- Also decreased the image margin to `1rem`

![image](https://user-images.githubusercontent.com/34704796/193829304-35ea0255-cda7-45f8-917c-faaff7f5b8ed.png)